eng/Build.props

```diff
@@ -1,45 +1,83 @@
 <Project>
 
+  <PropertyGroup>
+    <SubsetToLower>$(Subset.ToLowerInvariant())</SubsetToLower>
+  </PropertyGroup>
+
   <!--
     Projects for Arcade to build.
 
+    Supports '/p:Subset=<desired subset string>' dev build argument. If not set, all projects run.
+    Otherwise, each subset only runs if it is contained in the desired subset string. Each subset
+    listed in the arg should be separated by a '-' to avoid unintended matches.
```

Comment:
I'm not following how a `-` avoids unintended matches?  The matching logic is somewhat _fuzzy_ because of the use of `contains`.  Aren't there are a number of scenarios which could have unintended matches?  e.g. 'mismanaged`  I'm not saying there is anything wrong with that but the comment is somewhat misleading.

Reply:
Yeah, this should say *some* unintended matches, and even then I don't have any practical examples. This is specifically about two subset names combining to form some unexpected name.

So, say, `corehost` and `estimate`  are `corehostestimate`, which contains `test`.

But if I can make the validation better (your other comment) this will become a requirement rather than just a convention.